src/trio/_tools/gen_exports.py

```diff
@@ -184,8 +179,9 @@ def run_linters(file: File, source: str) -> str:
       SystemExit: If either failed.
     """
```

Comment:
Nit, grammar also needs updating, `either` referrers to two, but now it's only one.